README.md

```diff
@@ -49,12 +49,11 @@
 SpiceDB is the most mature open source project inspired by Google's internal authorization system: [Zanzibar].
 
 As of 2021, [broken access control became the #1 threat to web security according to OWASP][owasp].
-With SpiceDB, platform teams are armed with the same techniques for stopping this threat that the hyperscale tech companies have been doing for years behind closed doors.
+With SpiceDB, platform and product teams can be be protected by answering this question easily: "can subject X perform action Y on resource Z?".
 
-Similar to a relational database, developers define a schema, write data to the database, and query that data in various ways.
-However, unlike relational databases that use general-purpose SQL, SpiceDB exposes a gRPC API specifcally optimized for authorizing actions in your systems.
+Similar to a relational database, developers define a **schema**, write data in the form of **relationships**, and then use SpiceDB's clients to issue **permission checks** in their application to determine what actions a user can take on a resource. Other queries are also possible, such as "What can `subject` do?" or "Who can access `resource`?".
```

Comment:
might be worth saying `"On what can `subject` perform an action?" (LookupResources)` and `"Who can access `resource`? (LookupSubjects)"`

Reply:
At this point of the document, I think it's too early to introduce nomenclature of APIs. Right now, the focus is _what can this thing do_? and in later sections we can over _how?_